README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+    - semantic HTML is using tags that describe what is within. Instead of using just "div", use "dive class = '' "
```

Comment:
Semantic HTML gives meaning to the content and makes it easier for Search Engines, Screen Readers, and the browsers to know what the content is.if  you have a title, then wrapping it with the <h1> tag will let the browser know that it is a title. "class" is just a style attribute and doesn't define the content. I know there's is a lot to learn during this first week but as you keep practicing it will become second nature.